examples/deepspeed-multinode/sky.yaml

```diff
@@ -18,6 +24,7 @@ resources:
   # accelerators: A100-80GB:1  # Azure, GCP, SCP
   # accelerators: A10G:1  # AWS. Will OOM for (1) single_node/run_1.3b_lora.sh (2) multi_node/run_66b.sh.
   # accelerators: T4:1  # AWS, Azure, GCP. Will OOM for (1) single_node/run_1.3b_lora.sh (2) multi_node/run_66b.sh.
+  # image_id: docker:nvidia/cuda:12.1.1-devel-ubuntu20.04  # Use this image if running on Kubernetes
```

Comment:
Wondering if we should always use the docker image for consistency. Wdyt?

Reply:
I recall running into some issues with multi-node setup when trying to run with this image on GCP (kubernetes worked fine), thus only recommending for k8s for now.